CPP17.md

```diff
@@ -150,6 +150,17 @@ S x2 = S{123};        // mov eax, dword ptr [.L_ZZ4mainE2x2]
                       // .L_ZZ4mainE2x2: .long 123
 ```
 
+It can also be used to declare and define a member variable, such that it does not need to be initialized in the source file.
+```c++
+struct S {
+	S() : id(count++){};
+	~S(){count--;};
+
+	int id;
+	static inline int count{0}; // declare and initialize count to 0 within the class
+};
+```
```

Comment:
I've fixed up the formatting so it resembles the other code examples:
```c++
struct S {
  S() : id{count++} {}
  ~S() { count--; }
  int id;
  static inline int count{0}; // declare and initialize count to 0 within the class
};
```
Please use the above format.